internal/proxy/util.go

```diff
@@ -1792,11 +1792,20 @@ func verifyDynamicFieldData(schema *schemapb.CollectionSchema, insertMsg *msgstr
 			for _, rowData := range field.GetScalars().GetJsonData().GetData() {
 				jsonData := make(map[string]interface{})
 				if err := json.Unmarshal(rowData, &jsonData); err != nil {
-					return err
+					log.Warn("insert invalid dynamic data, milvus only support json map",
```

Comment:
incorrect user input should not cause a warn log? warn log is more like for internal system warns.

comment is not blocking, not requesting a change in this PR